content/subscription/details.md

```diff
@@ -8,15 +8,15 @@ This page provides an overview of each tier. To compare features available in ea
 
```

Comment:
@stephaurelio Can we expand the overview sentence  "This page provides an overview of each tier." ? Also a cross-reference to the Scout pricing page and vice versa?